MIP/mip-21/README.md

```diff
@@ -0,0 +1,91 @@
+
+To align with this approach and reduce complexity:
+
+- **Bridge Relayer as Attestor**: The Bridge Relayer in our system will function as an attestor, operating via a multisig scheme (i.e. the attestor collects signatures from a threshold number of comittee members that approve of the message). This relayer, through aggregated signatures, produces a proof on a signed message.  
+
+- **User Interaction**: Users, or other parties interested in completing the bridge, retrieve their funds on the target chain by submitting the signature and message.
+
+- **Validation Mechanism**: The off-chain signature produced by the multisig scheme is verified against the message on-chain.
+The challenge is to make sure that the signature is only used once, but the beauty is that the signature is provided by a multi-party system that do not need to provide onchain signatures.
+Relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.  
+
+Or on Circle's own words:
+
+1. An on-chain component on source domain emits a message.
+2. Circle's off-chain attestation service signs the message.
+3. The on-chain component at the destination domain receives the message, and forwards the message body to the specified recipient.
+
+This approach minimizes the need for additional components and avoids the reliance on synthetic assets, leading to a simplified and cost-effective bridge solution.
+
+### Flow
+
+1. **User Deposits**:  
+   - The user deposits tokens on the source chain, which triggers a deposit event.
+   
+2. **Bridge Relayer**:  
+   - The Bridge Relayer, listening for the deposit event, picks up the message or messages associated with the deposit.
+   
+3. **Multisig Signing**:  
+   - The multisig party signs the bridge transaction. The combination of signatures produces an aggregated signature that ties the signatures to the message.
+
+4. **Target Chain Verification**:  
+   - On the target chain, the aggregated signature and the message are used to verify the validity of the bridge transaction. The withdrawal can only occur if the multisig-provided signature is valid.
+
+5. **User Withdrawal**:  
+   - Once the signature is verified, the user can withdraw the corresponding funds on the target chain.
+
+This process is bi-directional, allowing funds to be bridged between Layer 1 and Layer 2, and vice versa, with minimal complexity and friction.
+
+## Reference Implementation
+
+For more details on how this model operates, refer to USDC's Cross-Chain Transfer Protocol (CCTP) documentation:  
+[USDC CCTP Overview](https://developers.circle.com/stablecoins/docs/generic-message-passing)
+
+## Verification
+
+### 1. **Correctness**: 
+The correctness of this proposal can be established by leveraging the success of CCTP, which has been operational and widely adopted.
```

Comment:
Regarding correctness, I don't think any success of CCTP makes sense to use as a reference. 

Nor is CCTP's implementation a reference implementation in an MIP. A reference implementation would mean an implementation for Movement's bridge.

I would change this so it either consistently mentions CCTP as an implementation from which this MIP draws inspiration, or include some code as a reference implementation for Movement's bridge. The key details have to do with how the Solidity modules and Move contracts would need to be modified, along with the bridge infrastructure.

Reply:
Not sure if I understood, I thik that this would be a major overhaul. But at least we would bootstrap our security with a different model. Major work would be the logic on the move modules.

Reply:
I support @andygolay in his statement. I think the MIP in its current form would classify as an MD, unless further details go into this about how specifically the components would be implemented. 

@l-monninger you are the MD master, is this content what you had in mind for an MD?

Reply:
@0xPrimata I have moved it over to MD type. are you ok with this change?